great_expectations/expectations/metrics/column_map_metrics/column_values_in_set.py

```diff
@@ -19,6 +20,14 @@
 except ImportError:
     sa = None
 
+try:
+    import polars as pl
+    from polars import DataFrame as pl_DataFrame
+
+except ImportError:
+    pl_DataFrame = None
+    pl = None
```

Comment:
Looks like you can use the import manager for polars and sqlalchemy, mind switching? Might help to centralize the import logic.

Reply:
great catch. I'll do it now